### ISSUE TITLE
A11y_SQLdatabase projects extension_Publish_Existing SQL Server_Voiceover: Voiceover is not announcing the dialog name once "Publish" dialog is opened.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/azuredatastudio#22160]
##GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #WCAG1.3.1; #MAC; #AINotInScope; #Info and Relationships; #Voiceover; #Element:dialog;

##Actual Result:​
Voiceover is not announcing the dialog name once "Publish" dialog is opened.
###Note:
Similar issue is observed for Advance option dialog.

##Expected Result:​
Voiceover should announce the dialog name once "Publish" and "Advance Publish Option" dialog is opened.

##Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

##Repro Steps:
1.Start Voiceover with using shortcut key Command+F5.
2.Launch Azure Data Studio
3.Tab till extension and search for “SQL Database Projects” and install it.
4.Tab till database project and press enter.
5“Database Project” panel will open.
6.Tab till existing project name and press context menu button(right click).
7.Tab till "Publish" menu-item in list and press enter.
8.Verify whether voiceover is announcing the dialog name as "Publish" dialog is opened or not.
##User Impact:
Voiceover user will miss the information about the dialog box name which will get open if voiceover is not announcing it's name.

##Attachment:
![publish -dialog name](https://user-images.githubusercontent.com/91586101/223082541-2e6b41e2-0f30-4aef-94f7-bd81214bacc4.gif)
Note issue :
![advance publish - dialog name](https://user-images.githubusercontent.com/91586101/223082412-e621a92a-39be-408d-8fa3-100f1dcc2246.gif)
